lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -91,6 +94,9 @@ def call(env)
           request_span.set_error(e) unless request_span.nil?
           raise e
         ensure
+          env[Ext::RACK_ENV_REQUEST_SPAN] = previous_request_span if previous_request_span
+          env[:datadog_rack_request_span] = env[Ext::RACK_ENV_REQUEST_SPAN]
```

Comment:
I understand the above line restores the first Rack span as the current Rack span, but what does `env[:datadog_rack_request_span]` have? What's it used for?

Also, I don't think we're supposed to put symbols in the Rack env as this will break other libraries that expect all keys to be strings. Should probably convert this to a string.